### ISSUE TITLE
Feedback from wokrshop:

### ISSUE THREAD

**Issue by yuliaUU** [kvarada/DSI-ML-workshop-2024#11]
Overall- great! especially all the examples shown and activities.

My feedback:

- include slide to define features vs target vs labels. so that it is more clear to everyone. clarify whether continuous target has labels or not, relate it to terms dependent and independent variables 
- decision trees: slide iterating the procedure: in shown image it was not clear what value=[0,6] stands for. what those numbers mean?
- slide :What’s going on? train_error and test_error graph: depth of decision tree was not introduced properly so x axis meaning was not clear. slide need a summary statement
- When you say Linear Classifier: it was not clear what that means: ie what classifier mean?
- I compiled some very nice resources that explain teh concepts in interactive way or just very good at explaining everything: may be we can add links as a last slide in teh workshp?
	- More on Overfitting: https://www.geeksforgeeks.org/underfitting-and-overfitting-in-machine-learning/
	- More about  Train, Test, and Validation Sets: https://mlu-explain.github.io/train-test-validation/.
	- More about Cross-Validation: https://mlu-explain.github.io/cross-validation/.
	- Bias -variance tradeoff: https://mlu-explain.github.io/bias-variance/
	- More about Decision Trees: https://mlu-explain.github.io/decision-tree/.
	- Random Forest: https://mlu-explain.github.io/random-forest/
	- k-means clustering algorithm:  https://youtu.be/4b5d3muPQmA
	- PCA method in mode details: https://www.youtube.com/watch?v=HMOI_lkzW08&list=RDCMUCtYLUTtgS3k1Fg4y5tAhLbw&index=8 OR StatQuest: https://www.youtube.com/watch?v=FgakZw6K1QQ&list=RDCMUCtYLUTtgS3k1Fg4y5tAhLbw&index=4
	- Bag of words: https://letsdatascience.com/bag-of-words/
	- More about Neural Networks https://mlu-explain.github.io/neural-networks/
	- Free course on NLP: https://www.nlpdemystified.org/

- each slide should have one sentence summary of the main ideas. especially the one showing graphs or outputs from the code.
- Would be great if you can advertise a course DSI designed for Micro-certificate "Health Data Analytics: Opportunities and Applications". Course 3 from this microcertificate, called "Health Data Analysis and Machine Learning", is designed for people with no coding experience and who want to learn ML technique. Health Data Analysis and Machine Learning next offering Nov 19, 2024. More info can be found here: https://extendedlearning.ubc.ca/courses/health-data-analysis-machine-learning/0116.
